_pkgdown.yml

```diff
@@ -18,7 +18,6 @@ navbar:
       href: https://pharmaverse.slack.com/archives/C030EB2M4GM
       aria-label: slack
 
-
 reference:
 - title: The six core xportr functions
 - contents:
```

Comment:
The pkgdown yaml config needs to be linted. I use prettier which auto-linted this for me. I can commit this change if you don't mind.

Reply:
I pushed the linted file now.
No, Our linter CI only lints R files because we only do `lintr::lint_package()` may be an overkill to check linting for `yaml` files just for one file :)